### PR TITLE
feat: CEC in M365 Copilot: enable feedbackloop and zero prompt(TS&JS)

### PR DESCRIPTION
**Pull request by QinghuiMeng-M**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/29180531/

### CODE REVIEW

File: templates/js/custom-copilot-rag-customize/src/app/app.js.tpl [hunk 1]
```diff
@@ -53,4 +54,9 @@ app.conversationUpdate("membersAdded", async (turnContext) => {
   }
 });
 
+app.feedbackLoop(async (context, state, feedbackLoopData) => {
+  //add custom feedback process logic here
+  console.log('Your feedback is ' + JSON.stringify(context.activity.value));
```
Comment (kuojianlu) [inline templates/js/custom-copilot-rag-customize/src/app/app.js.tpl:59]: Use double quotes.

File: templates/js/custom-copilot-assistant-assistants-api/appPackage/manifest.json.tpl [hunk 2]
```diff
@@ -29,7 +29,24 @@
...
-            "isNotificationOnly": false
+            "isNotificationOnly": false,
+            "commandLists": [
+                {
+                    "scopes": [
+                        "personal"
+                    ],
+                    "commands": [
+                        {
+                            "title": "Solve the equation: 3x + 11= 14",
```
Comment (kuojianlu) [inline templates/js/custom-copilot-assistant-assistants-api/appPackage/manifest.json.tpl:40]: All commands are tested and workable?

File: templates/js/custom-copilot-rag-customize/src/app/app.js.tpl (hunk 1 above)
Comment (QinghuiMeng-M) [reply templates/js/custom-copilot-rag-customize/src/app/app.js.tpl]: done

File: templates/js/custom-copilot-assistant-assistants-api/appPackage/manifest.json.tpl (hunk 2 above)
Comment (QinghuiMeng-M) [reply templates/js/custom-copilot-assistant-assistants-api/appPackage/manifest.json.tpl:40]: all tested, only “What's weather in redmond, us?” bot can't get the answer
image

Comment (kuojianlu) [reply templates/js/custom-copilot-assistant-assistants-api/appPackage/manifest.json.tpl:40]: Update to a workable prompt.

Comment (kuojianlu) [reply templates/js/custom-copilot-assistant-assistants-api/appPackage/manifest.json.tpl:40]: "Tell me the weather of San Francisco" is a possible workable prompt.

Comment (QinghuiMeng-M) [reply templates/js/custom-copilot-assistant-assistants-api/appPackage/manifest.json.tpl:40]: ok, but as the length limit, update command to "The weather of San Francisco"

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**